src/components/multi-select/__snapshots__/multi-select.test.jsx.snap

```diff
@@ -0,0 +1,77 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`<MultiSelect> has defaults 1`] = `
+<body>
+  <div>
+    <div
+      class="container"
+      id="test-id"
+    >
+      <div
+        role="presentation"
+      >
+        <label
+          class="label"
+          for="test-id-autocomplete"
+          id="test-id-label"
+        >
+          test label
+        </label>
+        <div
+          class="control-container"
+        >
+          <div
+            class="form-control-children-container"
+            role="presentation"
+          >
+            <div
+              aria-labelledby="test-id-label"
+              class="tag-list"
+              role="grid"
+            >
+              <input
+                aria-autocomplete="list"
+                aria-controls="test-id-listbox"
+                aria-describedby="test-id-empty"
+                aria-expanded="false"
+                aria-haspopup="listbox"
+                aria-labelledby="test-id-label"
```

Comment:
We don't necessarily need this. A native form control is labelled by `<label for="...">` or `aria-label` or `aria-labelledby`. I think the rule of thumb is to use either native or `aria` stuff and never both? I'm not sure tbh but I think simpler code the better.

Reply:
Cool, I'll remove this attribute.